hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakingRewardsHelper.java

```diff
@@ -145,6 +176,13 @@ public void increasePendingRewardsBy(final WritableNetworkStakingRewardsStore st
         final var stakingRewards = stakingRewardsStore.get();
         final var copy = stakingRewards.copyBuilder();
         stakingRewardsStore.put(copy.pendingRewards(newPendingRewards).build());
+
+        // Update the individual node pending node rewards
+        return currStakingInfo
+                .copyBuilder()
+                .unclaimedStakeRewardStart(0)
```

Comment:
⚠️ Shouldn't this only be reset to `0` at the end of a period? (C.f. mono-service [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ledger/accounts/staking/EndOfStakingPeriodCalculator.java#L148).)

Reply:
This is called only at the end of the period Michael 🤔